Update/tara_002.txt

```diff
@@ -1432,14 +1432,14 @@ void main()
 			NULL, "\"There, there.", Line_WaitForInput);
 	ModPlayVoiceLS(4, 5, "ps3/s17/05/110500029", 256, TRUE);
 	OutputLine(NULL, "……圭一、自信がないからと腕力に訴えるのはずるいのですよ」",
-			NULL, " ...Keiichi is only harassing you because he's not confident in his own capabilities.\"", Line_Normal);
+			NULL, " ...Keiichi, it's not fair to resort to violence because you're not confident in your own capabilities.\"", Line_Normal);
```

Comment:
```suggestion
			NULL, " ...Keiichi, it's not fair to resort to violence just because you're not confident in your own capabilities.\"", Line_Normal);
```